### PR TITLE
Headline section design

### PR DESCRIPTION
**Pull request by aimalamiri**
Headline section design work is completed

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        </ul>
+    </nav>
+    <header class="header">
+        <div class="social-media-links">
+            <a href="#" class="btn btn-circle"><img src="assets/svg/github.svg" alt="Github icon"></a>
+            <a href="#" class="btn btn-circle"><img src="assets/svg/linkedin.svg" alt="Linkedin icon"></a>
+            <a href="#" class="btn btn-circle"><img src="assets/svg/angel-list.svg" alt="Angel List icon"></a>
+            <a href="#" class="btn btn-circle"><img src="assets/svg/twitter.svg" alt="Twitter icon"></a>
+            <a href="#" class="btn btn-circle"><img src="assets/svg/medium.svg" alt="Medium Icon"></a>
+        </div>
```
Comment (micheaol) [inline index.html]: Please ensure that the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has `a link (a tag)` according to the requirement.

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+  padding: 26px 0;
+  box-sizing: border-box;
+  justify-content: space-between;
+}
+
+.navbar-logo {
+  font-size: 20px;
+  line-height: 20px;
+  color: #fff;
+}
```
Comment (micheaol) [inline styles.css:22]: Please ensure that you stick to the design as much as possible (e.g. font, colors, images, tests, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Your Implementation | Figma Design
:---:|:---:
 
your_design | figma_design

File: index.html [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        <section class="header-content container">
+            <h1 class="header-title font-crete-round text-orange">Hey There. I'm Aimal</h1>
+            <h2 class="header-subtitle text-white font-inter">I’m a Software Developer</h2>
+            <p class="header-text text-white font-inter">I can help you build a product , feature or website Look
+                through some of my work and experience! If you like what you see and have a project you need coded,
+                don’t hestiate to contact me.</p>
+        </section>
+        <img class="header-image" src="./assets/png/header-illustration.png" alt="Header illustration Photo">
```
Comment (micheaol) [inline index.html]: Please ensure that you use `css` background image (decorations are always background images, not `img` tags)

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @aimalamiri , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 

## Required Changes ♻️

- _Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (aimalamiri) [reply index.html]: Thanks for mentioning, it's fixed now.

File: styles.css (hunk 2 above)
Comment (aimalamiri) [reply styles.css:22]: The page is now responsive, and the sizing is now the same as Figma design, but the font which is used in Figma is called "Crete Round" and I used the same font from google, but it looks a bit different.
I checked and the font is applied on text but it looks a bit different, and I don't know why?

File: index.html (hunk 3 above)
Comment (aimalamiri) [reply index.html]: That's a good point, and I fixed it now.

File: styles.css (hunk 2 above)
Comment (aimalamiri) [reply styles.css:22]: Oh I forget to mention you @micheaol 😊

**Review by mke2111 — COMMENTED** [review]
Hello @aimalamiri  👋,

_This is an additional review as a result of the second opinion request._

Great work you have achieved in this milestone. 💯 

I can see all the changes you have applied as suggested by the first reviewer. Kindly submit another CRR so that another reviewer can approve your request if all changes have been implemented. 

> Only submit for a second opinion if you have something like a request that is not clear, or if you disagree with the changes requested.

Feel free to request another code review request (CRR) whenever you are ready. Do this through your dashboard. If you need help with this or further clarification. Please tag me in this thread just in case.

Keep rocking 🚀!

**Comment by aimalamiri** [comment]
Hi @mke2111 thanks for your comment, I sent a CRR and waiting for my code review.

**Review by Donard97 — APPROVED** [review]
## Hi @aimalamiri , 👋  

You did an amazing job completing all requirements from the other reviewer. 💯 💯 

**Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:** 

## Congratulations! 🎉 🎉 

giphy (11)

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**